src/style.css

```diff
@@ -0,0 +1,88 @@
+
+@font-face {
+  font-family: "Avenir-book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.searchBar {
+  margin-bottom: 20px;
+}
+
+[data-qa="big"] {
+  margin-top: 20px;
+}
```

Comment:
you have a class for this element, use class for stylization 
Try not to stylize elements by their arguments